tests/ui/single_char_pattern.stderr

```diff
@@ -102,5 +102,11 @@ error: single-character string constant used as pattern
 37 |     x.trim_right_matches("x");
    |     ---------------------^^^- help: try using a char instead: `x.trim_right_matches('x')`
 
-error: aborting due to 17 previous errors
+error: single-character string constant used as pattern
+  --> $DIR/single_char_pattern.rs:39:13
+   |
+39 |     x.split("/n");
```

Comment:
huh why is this a forward slash here

Reply:
The output was correct but the test required this.

Reply:
As I understand it, this is required due to compiletest normalization.
https://github.com/rust-lang/rust/blob/master/src/test/COMPILER_TESTS.md#normalization
It looks like it does that due to path differences on Windows. Yes, it's weird.